add module 1.1 baseline (#5)
PeARSearch/PeARS-fruit-fly: datasets/README.md

# The dataset directory

This directory contains the datasets to be processed by the fruit fly. The **prepare_datasets** notebook contains the
procedures to create the ready-to-train datasets, including:

* Select the classes and download articles from Wikipedia.

* Tokenize and split the Wikipedia dataset, 20newsgroups dataset, and Web of Science dataset.

## The Wikipedia dataset

> Please download the directory **wikipedia** in this Google Drive:

Reviewer: ```suggestion
Please download the directory **wikipedia** from this URL:
```

> https://drive.google.com/drive/folders/1R_BBCdLH7FRdyD071GRteigY6ZztUN5z?usp=sharing

Reviewer: http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia.zip

> https://drive.google.com/drive/folders/1R_BBCdLH7FRdyD071GRteigY6ZztUN5z?usp=sharing

Reviewer: [http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia.zip](http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia.zip)

> https://drive.google.com/drive/folders/1R_BBCdLH7FRdyD071GRteigY6ZztUN5z?usp=sharing

Reviewer: ```suggestion
[http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia.zip](http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia.zip)
```

> and place it in this **datasets** directory. If you need the un-tokenized version, please download **wiki_dataset_raw**.

Reviewer: ... please download http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia-raw.zip

> and place it in this **datasets** directory. If you need the un-tokenized version, please download **wiki_dataset_raw**.

Reviewer: ```suggestion
and place it in this **datasets** directory. If you need the un-tokenized version, please download [the raw dataset](http://pearsproject.org/static/datasets/pears-fruit-fly-wikipedia-raw.zip).
```

> You may follow the first section of the **prepare_datasets** notebook to understand the procedure that create the dataset.

Reviewer: ```suggestion
You may follow the first section of the **prepare_datasets** notebook to understand the procedure that creates the dataset.
```
